albatross/models/ransac.h

```diff
@@ -229,6 +229,8 @@ class GenericRansac : public RegressionModel<FeatureType> {
     RegressionDataset<FeatureType> inliers =
         ransac(dataset, fold_indexer, sub_model_, metric_, inlier_threshold_,
                random_sample_size_, min_inliers_, max_iterations_);
+    this->metadata_["post_ransac_feature_count"] =
```

Comment:
And `input_feature_count` is the `pre_ransac_feature_count`?

Reply:
yeah, I just didn't want to call it that since ransac isn't necessarily run every time.